examples/seismic/self_adjoint/time_blocking.ipynb

```diff
@@ -0,0 +1,1365 @@
+{
```

Comment:
<p>The forward linear modeling operator is nonlinear in <code>m</code>  and linear in both <code>q</code>  and <code>s</code>  , but <code>s</code>  is missing in the symbol/expr</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Yikes, made me realize almost all the other nb are messed up. Damn slim cats caused a notation revolution. Ill do those in a separate PR.

This one is fixed.